src/test/java/no/nav/foreldrepenger/common/util/ForeldrepengerTestUtils.java

```diff
@@ -171,9 +172,9 @@ public static Ettersending ettersending() {
     }
 
     private static List<Tilrettelegging> tilrettelegging(String... vedleggRefs) {
-        return Lists.newArrayList(helTilrettelegging(vedleggRefs), helTilrettelegging(vedleggRefs),
-                delvisTilrettelegging(vedleggRefs),
-                ingenTilrettelegging(vedleggRefs));
+        return Stream.of(helTilrettelegging(vedleggRefs), helTilrettelegging(vedleggRefs),
```

Comment:
Er det forskjell på dette og List.of?

Reply:
List.of blir immutable
Lists.newArrayList fra google lager en mutable arraylist
Stream.of().collect(Collectors.toList()) lager en mutable arraylist (en måte å gjøre new arraylist. add add one-liner)

Reply:
Stream.of().collect(Collectors.toUnmodifiableList()) er det samme som List.of()